bin/pytorch_inference/evaluate.py

```diff
@@ -297,13 +317,32 @@ def test_evaluation(args):
             print('SUCCESS: inference results match expected', flush=True)
             print()
 
+def threading_benchmark(args):
+    threading_options = [1, 2, 3, 4, 8, 12, 16]
+    results = []
+    for libtorch_threads in threading_options:
+        for mkl_threads in threading_options:
```

Comment:
I can't see where `mkl_threads` is used apart from the `print` statements

Reply:
ah yes, forgot to remove that